BitmovinAnalyticsCollector/Classes/Collector/util/Util.swift

```diff
@@ -83,6 +83,21 @@ class Util {
         }
         return codecs;
     }
+    
+    static func getStreamTypeFromUrl(url: String) -> StreamType? {
```

Comment:
Not very Swift-y naming. I would name this function `streamType(from url: string) -> StreamType?`